source/Private/New-GuestConfigurationPolicyContent.ps1

```diff
@@ -71,12 +75,41 @@ function New-GuestConfigurationPolicyContent
         Parameter = $Parameter
     }
 
+    if ($ContentManagedIdentity)
+    {
+        $metadataSectionParameters.ContentManagedIdentity = $ContentManagedIdentity
+    }
+
     $metadataSection = New-GuestConfigurationPolicyMetadataSection @metadataSectionParameters
 
     $parametersSection = New-GuestConfigurationPolicyParametersSection -Parameter $Parameter
 
     $conditionsSection = New-GuestConfigurationPolicyConditionsSection -Platform $Platform -Tag $Tag -IncludeVMSS $IncludeVMSS
 
+    if ($ContentManagedIdentity)
+    {
+        foreach ($anyOf in $conditionsSection.anyOf)
```

Comment:
This parameter should be passed through to the conditions section and parameters section cmdlets rather than removed afterwards.

Reply:
Passed it in.